docs/javascript/v2/guides/javascript-quickstart.mdx

```diff
@@ -3,154 +3,306 @@ title: JavaScript Quickstart Guide
 
-TL;DR - You can find the link to the complete example over [here](https://codesandbox.io/s/hms-js-quickstart-5rmes). There is also a version of the quickstart using CDN [here](https://codepen.io/triptu/pen/dymxmoR?editors=0010).
+TL;DR - Try out this complete example in [CodeSandbox](https://codesandbox.io/s/hms-js-quickstart-5rmes).
 
-### Prerequisites
+## Setup a local Javascript app
 
-To get started you must be familiar with rudimentary JavaScript and have [nodejs](https://nodejs.org/) installed on your machine.
+In this walk-through, we will use [ParcelJS](https://parceljs.org/getting-started/webapp/) a web application bundler that requires zero configuration. If you wish to use any other library or framework, you can set up your app using that.
 
-### Installing the dependencies
+### Initialize a project
+
+You can initialize a project using `npm init` or `yarn init`. While initializing the project you can enter the details for your project as you wish.
+
+### Install ParcelJS & 100ms SDK
+
+Then, install ParcelJS and 100ms SDK into your app using yarn or npm like below:
 
 ```bash section=InstallingTheDependencies sectionIndex=1
-npm install --save @100mslive/hms-video-store@latest
+yarn add --dev parcel @100mslive/hms-video-store
 ```
 
-### Initializing the SDK
+or
 
-Let us start with initializing the libraries. We need an instance of `HMSStore` and `HMSActions` to get started. Copy the code snippet below and paste it into your project.
+```bash section=InstallingTheDependencies sectionIndex=2
+npm install --save-dev parcel @100mslive/hms-video-store
+```
+
+### Project setup
 
-```jsx:script.js section=InitializingTheSDK sectionIndex=1
-import { HMSReactiveStore } from '@100mslive/hms-video-store';
+Now that ParcelJS is installed, let's create basic source files for our video-conferencing application. Let’s create the files and folders in the below manner:
 
-const hms = new HMSReactiveStore();
-const hmsStore = hms.getStore();
-const hmsActions = hms.getHMSActions();
+```
+ParcelJS-test
+  - index.html
+  - src
+    - index.js
+    - styles.css
 ```
 
-`hmsStore` will hold the complete state of the application such as details of all the participants. We can also visualize this state at any time using the
-[devtools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en) extension.
+## Add basic functionalities
 
-`hmsActions` will help us perform actions such as joining the room, mute our audio and send messages.
+In this section, we will walk you through instructions to add basic functionalities for a video conferencing application so that you can test the app locally to see how two or more peers can join this app to communicate with each other.
 
 ### Concepts
```

Comment:
we should link to the basic concepts page from here

Reply:
done